bundle/Resources/config/routing.yaml

```diff
@@ -0,0 +1,5 @@
+netgen.better_ibexa_admin_ui.content.update_always_available:
+    path: /content/update-always-available
+    methods: ['POST']
+    defaults:
```

Comment:
this can be written as

```
controller: 'Netgen\Bundle\BetterIbexaAdminUIBundle\Controller\Content\UpdateAlwaysAvailable'
```

So just `controller` key instead of `defaults._controller`

Reply:
Updated in https://github.com/netgen/better-ibexa-admin-ui/pull/4/commits/02a65c0ecf43b6b0be45337ff0fdd249bd8e4f47.